---
monitor:
  enabled: true
  print_hz: 1.0
  ewma_tau_sec: 2.0
  warn_grace_sec: 2.0
  use_ansi_color: true

bags:
  - name: "main2"
    bag_uri: "/tmp/bag_test"
    storage_id: "mcap"
    write_frequency_hz: 1.0
    max_queue_size: 30000
    uri_with_datetime: true
    topics:
      - name: "/imu/data"
        type: "sensor_msgs/msg/Imu"
        qos: "default"
        mode: "generic"
        monitor:
          expected_hz: 100.0
          warn_below_hz: 95.0
      - name: "/left_camera/compressed"
        type: "sensor_msgs/msg/CompressedImage"
        qos: "default"
        mode: "generic"
        monitor:
          expected_hz: 10.0
          warn_below_hz: 9.0
      - name: "/velodyne_points"
        type: "sensor_msgs/msg/PointCloud2"
        qos: "default"
        mode: "generic"
        monitor:
          expected_hz: 10.0
          warn_below_hz: 9.0
